device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/InternalClient.java

```diff
@@ -180,6 +183,8 @@ public void closeNow() throws IOException
      */
     public void sendEventAsync(Message message, IotHubEventCallback callback, Object callbackContext)
     {
+        verifyNotUnregisteredIfMultiplexing();
```

Comment:
double negative method name. We can just call it `verifyRegisteredIfMultiplexing`